src/Components/SuperAdminsForm/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import styles from './super-admins-form.module.css';
+
+function SuperAdminsForm() {
+  const url = window.location.href;
+  const id = url.substring(url.lastIndexOf('=') + 1);
```

Comment:
add a new const here
```suggestion
  const id = url.substring(url.lastIndexOf('=') + 1);
  const formMode = id ? 'edit' : 'create';
```